README.md

```diff
@@ -121,6 +122,21 @@ tags.create(name="Tag", slug="tag", api_version="1.2",)
 tags.list(api_version="1.3",)
 ```
 
+### Retry logic
+
+By default, the client will not retry any operation. This behavior can be adjusted via the `retries` optional parameters. This will only affect for HTTP codes: 429, 501, 502, 503 and 504.
```

Comment:
```suggestion
By default, the client will not retry any operation. This behavior can be adjusted via the `retries` optional parameters. This will only affect for HTTP codes: 429, 500, 502, 503 and 504.
```